package.yaml

```diff
@@ -25,6 +25,8 @@ dependencies:
 
 library:
   source-dirs: src
+  dependencies:
```

Comment:
You should be able to drop `base64-bytestring` and `free` from global deps too.

Reply:
Now you know why I mentioned weeder being broken... ;)